apps/Unciv/install-32

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+### make directory ~/Unciv if not available
+if [[ ! -d ~/Unciv ]]; then
+  mkdir ~/Unciv || error 'Failed to make folder $HOME/Unciv!'
+fi
+
+### Add LICENSE from Github if doesn't exist
+if [[ ! -f ~/Unciv/LICENSE ]]; then
+  wget -O ~/Unciv/LICENSE ~ https://raw.githubusercontent.com/yairm210/Unciv/master/LICENSE || error 'Failed to make folder $HOME/Unciv!'
+fi
+
+
+### Download Zulu OpenJDK 8.58.0.13 and Extract JRE from it
+### Link: https://www.azul.com/downloads/?os=linux&architecture=arm-32-bit-hf
+
+# the package 'default-jre' can't be used as it takes too much ram
+# thus using Zulu is better as it performs much much smoother
+# with 'default-jre' the game is hardly playable
+
+## if ~/Unciv/jre doesn't exist
+if [[ ! -d ~/Unciv/jre ]]: then
+  # -c countinue, -P directory
+  wget -cP ~ https://cdn.azul.com/zulu-embedded/bin/zulu8.58.0.13-ca-jdk8.0.312-linux_aarch32hf.tar.gz || error 'Failed to download jre for Unciv!'
+
+  # Extract jre folder
+  tar --overwrite -xvf ~/zulu8.58.0.13-ca-jdk8.0.312-linux_aarch32hf.tar.gz --wildcards */jre || error 'Failed to ectract jre for Unciv!'
+
+  # move jre to ~/Unciv
+  mv ~/zulu8.58.0.13-ca-jdk8.0.312-linux_aarch32hf/jre ~/Unciv/jre || error "Failed to move jre for Unciv to $HOME/Unciv!"
+fi
+
+## remove artifacts
+if [[ -f ~/zulu8.58.0.13-ca-jdk8.0.312-linux_aarch32hf.tar.gz ]]; then
+  rm ~/zulu8.58.0.13-ca-jdk8.0.312-linux_aarch32hf.tar.gz || echo 'Failed to remove jre artifatcs! Skipping ...'
```

Comment:
You could simplify this by removing the `if` statement and using `rm -f`.

Reply:
Ok, yes, it works. Both for folder and file. Fixed in https://github.com/touhidurrr/pi-apps/commit/00638217a58f8323b8cd04dea85caed6463a73ed.